www/includes/easyparliament/templates/html/header.php

```diff
@@ -18,11 +18,12 @@
     <link rel="author" title="Send feedback" href="mailto:<?= str_replace('@', '&#64;', CONTACTEMAIL) ?>">
     <link rel="home" title="Home" href="http://<?= DOMAIN ?>/">
 
-    <meta property="og:title" content="TheyWorkForYou">
+    <meta property="og:site_name" content="TheyWorkForYou">
+    <meta property="og:title" content="<?= preg_replace('#<[^>]*>#', '', $page_title) ?>">
     <meta property="og:type" content="website">
     <meta property="og:url" content="http://<?= DOMAIN ?>">
     <meta property="og:image" content="http://<?= DOMAIN ?>/images/favicon-256.png">
-    <meta property="og:description" content="TheyWorkForYou is a website which makes it easy to keep track of your local MP's activities.">
+    <meta property="og:description" content="<?= _htmlentities($meta_description) ?>">
```

Comment:
I have just pushed a commit that should fix that, by adding a non-empty fallback in the Header.php class. https://github.com/mysociety/theyworkforyou/pull/1121/commits/013cee1b815b5d1381c61a273ad4728080816638